### DISCUSSION TITLE
Keyboard Timeout question

### DISCUSSION THREAD

**Discussion by SynStark** [seerge/g-helper#1210] [Ideas]
The Keyboard Timeout turning off the keyboard ligts if not used is one of my favorite features, but why does the keyboard lights up when I'm clicking or even moving the mouse? Can this be turned off as well, and light up only when pressing a key?

**Comment by seerge** [comment]
@SynStark its tied to windows native activity timer, and moving mouse (or touchpad) is also considered as activity by windows
